content/gsoc_application.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Gsoc_application"
+date: 2020-01-31T17:53:51Z
+draft: false
+---
+
+# Organisation profile
+
+## Descriptions 
+
+These descriptions will be displayed on the organization list page (Short Description) and on your organization's page (Long Description). [More details](https://developers.google.com/open-source/gsoc/help/org-profile#descriptions).
+
+The Long Description may include [limited Markdown](https://developers.google.com/open-source/gsoc/help/markdown).
+
+### Short description (180 characters)
+
+> Replace Firefox's default control mechanism with one modelled on the one true editor, Vim. Our Firefox add-on has about 10,000 users, most of whom are programmers.
+
+### Long description (2000 characters)
+
+> # A browser extension for programmers
+> 
+> Our extension for Firefox, Tridactyl, is used by about 10,000 users every day. It aims to make controlling Firefox more like controlling Vim, a popular text editor. We aim to make as much as possible within the browser programmable; our users have, for example, created keyboard shortcuts to clone a project they are viewing on GitHub and open a terminal in that directory.
+> 
+> Our users are generally skilled programmers who want to have more control over their browser.
+
+## Proposals
+
+Guidance for students on how to apply to your organization. Should include any prerequisites or requirements. You may wish to include a template or tips for their proposals. May include limited Markdown.
+
+Enter tags that students can select (one) from and apply to their own proposals to help organize them. Examples: New Feature, Optimization. You can also use these to designate "sub-organizations" if you are an umbrella organization. 
+
+### Application instructions (1500 characters)
+
+> TBC
```

Comment:
> The steps to apply are simple:
> 
> 1. Join our chatroom (Riot) and say hello, mentioning that you are thinking of applying to GSoC - people in this room will be happy to help you if you get stuck with any of the later steps
> 2. Get your development environment set up, build and run Tridactyl from source - instructions are on our GitHub page
> 3. Find an [easy bug](https://github.com/tridactyl/tridactyl/issues?utf8=%E2%9C%93&q=sort%3Aupdated-desc+label%3A%22good+first+issue%22+is%3Aopen) and try to fix it, then open a pull request. Even if you didn't manage to fix the bug, the pull request can be marked as a draft and we can help you out
> 4. Look at our ideas list and pick an idea or come up with one of your own
> 5. Start writing your draft proposal for GSoC. Ask us for feedback on it in Riot (e.g. via Google Docs or GitHub gist). A python project, mailman, has [some good advice](https://turnbull.sk.tsukuba.ac.jp/Blog/SPAM.txt#how-to-spam-in-detail) on writing a proposal. As a minimum, it should include:
>     
>     - An introduction, telling us about yourself and your background (what you are studying, previous experience you have, any URLs you can give us linking to your work)
>     - A brief summary of your proposal in your own words in a handful of sentences
>     - A sketch of the design for your proposal in a few sentences, perhaps with some pseudocode
>     - A schedule for what you want to have done by when - bonus points for referring to the [GSoC timeline](https://summerofcode.withgoogle.com/how-it-works/#timeline)
>         - these "milestones" should be fairly concrete, e.g. "tests are written" and "tests pass", not "code is about half finished"
> 
> 6. Send in your application

I'm finding it a little hard to work out how to do the "send in your application". It looks like organisations receive the applications and then send the ones they want to Google. It might be worth setting up an @ tridactyl-dot-xyz email address for the occasion.

Reply:
It looks like people apply via the GSoC website.